.settings/org.eclipse.m2e.core.prefs

```diff
@@ -0,0 +1,4 @@
+activeProfiles=
+eclipse.preferences.version=1
```

Comment:
it shouldnt be downloaded and seen by someone who uses different IDE than eclipse